src/test/README.md

```diff
@@ -1,3 +1,25 @@
+# Unit tests
+
+The sources in this directory are unit test cases. Boost includes a
+unit testing framework, and since Bitcoin Core already uses Boost, it makes
+sense to simply use this framework rather than require developers to
+configure some other framework (we want as few impediments to creating
+unit tests as possible).
+
+The build system is setup to compile an executable called `test_bitcoin`
```

Comment:
nit: a "setup" is a noun, "set up" is the verb version which should be used here, e.g. "The build system is set up to compile..."

Reply:
done